ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/PutBulk.java

```diff
@@ -107,29 +120,34 @@ public PutBulkResult call() throws Exception {
         /* Ensure the bucket exists and if not create it */
         helpers.ensureBucketExists(this.bucketName);
 
-        if (sync) {
-            if (!SyncUtils.isSyncSupported(getClient())) {
-                return new PutBulkResult("Failed: The sync command is not supported with your version of BlackPearl.");
-            }
+        if (pipe) {
```

Comment:
This is starting to have quite a bit of nesting.  Can this get broken out into different small functions.